packages/main/src/components/media-chat/LiveBroadcast.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import AgoraUIKit, { layout } from "agora-react-uikit";
+import "agora-react-uikit/dist/index.css";
+
+const LiveBroadcast = () => {
+  const [videocall, setVideocall] = useState(true);
+  const [isHost, setHost] = useState(true);
+
+  const rtcProps = {
+    appId: "9e95b753731148ff9f6ab866502fe3df",
+    channel: "live",
+    token:
+      "007eJxTYChtKGew0hY63SLCajbvL4Nj23lpM3Mb09Qzwv0eczyPNikwWKZamiaZmxqbGxsamlikpVmmmSUmWZiZmRoYpaUap6RZZzYlNwQyMtyaK8vEyACBID4LQ05mWSoDAwD7whta", //add your token if using app in secured mode
```

Comment:
stop committing api tokens
another member of your team has already changed this to use an ENV instead

Reply:
I've stored them in an env file